nixos/modules/system/boot/plymouth.nix

```diff
@@ -127,21 +156,29 @@ in
       mkdir -p $out/share/plymouth/themes
       cp ${plymouth}/share/plymouth/plymouthd.defaults $out/share/plymouth
 
-      # copy themes into working directory for patching
+      # Copy themes into working directory for patching
       mkdir themes
-      # use -L to copy the directories proper, not the symlinks to them
-      cp -r -L ${themesEnv}/share/plymouth/themes/{text,details,${cfg.theme}} themes
 
-      # patch out any attempted references to the theme or plymouth's themes directory
+      # Use -L to copy the directories proper, not the symlinks to them.
```

Comment:
Could you also mention why?

Reply:
It says it above, so I thought it made sense to not repeat it every time it's used

Reply:
extraUtils can't contain references to nix/store, and that's why we're patching. I thought that bit is usually a bit obvious whenever you're working with extraUtilsCommands